src/app/(pages)/layout.tsx

```
@@ -0,0 +1,73 @@
+    fetchEstablishments();
+  const { fetchMenuItems } = useNavigation();
+
+  useEffect(() => {
+    fetchMenuItems();
+  }, []);
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', width: '100%' }}>
+      <NavigationHeader />
+      {currentEstablishment &&
+        currentEstablishment.backgroundImage &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)